### PR TITLE
Refactoring images uploads

### PR DESCRIPTION
**Pull request by danymill**

### CODE REVIEW

File: client/app/_services/company-settings.provider.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
+export function CompanySettingsProvider() {
+    const local = {
+        companyLogo: {
+            name: 'COMPANY_LOGO_URL',
+            value: '',
+        }
```
Comment (DmitryMishin) [inline client/app/_services/company-settings.provider.js:6]: comma

File: client/app/_services/services.module.js [hunk 2]
```diff
@@ -13,8 +13,10 @@ import pageTitleService from './page-title/page-title.service';
 import observerService from './observer/observer.service';
 import logLevelService from './log-level.service';
 import fullScreenService from './full-screen/full-screen.service';
+import { CompanySettingsProvider } from "./company-settings.provider";
```
Comment (DmitryMishin) [inline client/app/_services/services.module.js:16]: single quotes

File: client/app/_user/user.controller.js [hunk 3]
```diff
@@ -3,8 +3,20 @@
...
+    UserService,
+    DashboardService,
+    UtilService,
+    authService,
+    appConfig,
+    $q,
+    $state,
+    messageService,
+    $rootScope,
+    pageTitleService
```
Comment (DmitryMishin) [inline client/app/_user/user.controller.js:18]: comma

**Review by dzmitrymishyn — CHANGES_REQUESTED** [review]

File: client/app/_services/company-settings.provider.js (hunk 1 above)
Comment (danymill) [reply client/app/_services/company-settings.provider.js]: fixed

File: client/app/_services/services.module.js (hunk 2 above)
Comment (danymill) [reply client/app/_services/services.module.js]: fixed

File: client/app/_user/user.controller.js (hunk 3 above)
Comment (danymill) [reply client/app/_user/user.controller.js]: fixed

**Review by dzmitrymishyn — APPROVED** [review]

**Pull request merged**